package/libs/libnl/patches/102-revert-build-enable-building-cli-during-tests.patch

```diff
@@ -37,18 +37,18 @@ Upstream status: https://github.com/thom311/libnl/pull/141
  
```

Comment:
The linked pull request mentions that this problem was also fixed some time ago:
 https://github.com/thom311/libnl/pull/141
This is probably also not needed any more